Uplift/Networking/NetworkManager.swift

```diff
@@ -118,6 +133,20 @@ struct NetworkManager {
         }
     }
 
+    func getGymClassesForDate(date: String, completion: @escaping ([GymClassInstance]) -> Void, failure: (() -> Void)?) {
+        apollo.fetch(query: TodaysClassesQuery(date: date)) { result, error in
+            guard error == nil,
+                let data = result?.data,
+                let classes = data.classes else {
+                    failure?()
+                    return
+            }
+            let gymClassInstances = classes.compactMap({ self.getGymClassInstance(from: $0) })
+
+            completion(gymClassInstances)
+        }
+    }
```

Comment:
shouldn't need to make this a new function either if the only difference is the optional `failure` argument